sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -3512,6 +3521,9 @@ class SQLConf extends Serializable with Logging {
 
   def disabledJdbcConnectionProviders: String = getConf(SQLConf.DISABLED_JDBC_CONN_PROVIDER_LIST)
 
+  def disabledStreamingCustomMetrics: String =
```

Comment:
nit: If there's only one place to get the conf, maybe we can directly get the conf on the caller side.

Reply:
Yeah, we can inline it. Removed.